tcms/testruns/models.py

```diff
@@ -243,39 +241,11 @@ class TestExecutionStatus(TCMSActionModel):
     IDLE = 'IDLE'
     WAIVED = 'WAIVED'
 
-    _icons = {
-        'IDLE': 'fa fa-question-circle-o',
-        'RUNNING': 'fa fa-play-circle-o',
-        'PAUSED': 'fa fa-pause-circle-o',
-        PASSED: 'fa fa-check-circle-o',
-        FAILED: 'fa fa-times-circle-o',
-        BLOCKED: 'fa fa-stop-circle-o',
-        'ERROR': 'fa fa-minus-circle',
-        WAIVED: 'fa fa-commenting-o',
-    }
-
-    _css_classes = {
-        IDLE: 'idle',
-        FAILED: 'fail',
-        PASSED: 'pass',
-        BLOCKED: 'fail',
-        'ERROR': 'fail',
-    }
-
-    _colors = {
-        IDLE: '#72767b',
-        'PASS': '#92d400',
-        'FAIL': '#cc0000'
-    }
-
-    complete_status_names = (PASSED, 'ERROR', FAILED, WAIVED)
-    failure_status_names = ('ERROR', FAILED, BLOCKED)
-    passed_status_names = (PASSED, WAIVED)
-    idle_status_names = (IDLE,)
-    chart_status_names = ('pass', 'fail', 'idle', 'other')
-
     id = models.AutoField(db_column='case_run_status_id', primary_key=True)
     name = models.CharField(max_length=60, blank=True, unique=True)
+    weight = models.IntegerField(name='weight', default=0)
+    icon = models.CharField(name="icon", max_length=60)
```

Comment:
- 64, use powers of 2
- do you really need the `name` attribute here ? If not remove it.